packages/core/src/Utils/pwa/register-service-worker.js

```diff
@@ -40,6 +42,7 @@ export default function register() {
                                     // the fresh content will have been added to the cache.
                                     // It's the perfect time to display a "New content is
                                     // available; please refresh." message in your web app.
+                                    if (performance.now() < AUTO_REFRESH_THRESHOLD) window.location.reload();
```

Comment:
the above comment doesn't relate to this line, shall we move it above the comments?

Reply:
:+1: